### PR TITLE
Avoid accessing java service interfaces in backbox tests

### PR DESCRIPTION
**Pull request by dimas-b**
This is in preparation to running test Stargate nodes in forked JVMs

### CODE REVIEW

**Review by dougwettlaufer — APPROVED** [review]

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/GraphqlTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6286296&bid=20454288)

File: testing/src/test/java/io/stargate/it/http/GraphqlTest.java [hunk 1]
```diff
@@ -85,9 +82,8 @@
  */
 @NotThreadSafe
 public class GraphqlTest extends BaseOsgiIntegrationTest {
-  private static final Logger logger = LoggerFactory.getLogger(GraphqlTest.class);
 
-  private DataStore dataStore;
+  private CqlSession session;
```
Comment (tomekl007) [inline testing/src/test/java/io/stargate/it/http/GraphqlTest.java:86]: That's a very nice idea to replace direct access with `CqlSession` +1 on that

File: testing/src/test/java/io/stargate/it/http/GraphqlTest.java [hunk 2]
```diff
@@ -98,88 +94,106 @@ public GraphqlTest(ClusterConnectionInfo backend) {
...
-        .column("customer_name", Type.Text, Kind.Clustering)
-        .column("id", Type.Uuid)
-        .column("prod_id", Type.Uuid)
-        .column("address", Type.Text)
-        .column("description", Type.Text)
-        .column("price", Type.Decimal)
-        .column("sell_price", Type.Decimal)
-        .execute();
-
-    dataStore.waitForSchemaAgreement();
```
Comment (tomekl007) [inline testing/src/test/java/io/stargate/it/http/GraphqlTest.java:152]: if we don't need the `dataStore.waitForSchemaAgreement();` anymore, then it is a strong reason to replace it with `CqlSession`

**Review by tomekl007 — APPROVED** [review]
LGTM but there are some Jenkins failures

File: testing/src/test/java/io/stargate/it/http/GraphqlTest.java (hunk 2 above)
Comment (mpenick) [reply testing/src/test/java/io/stargate/it/http/GraphqlTest.java:152]: This was never needed. `DataStore` automatically wait for schema agreement after DDL queries. I'm not sure why this was explicitly done.

**Review by mpenick — DISMISSED** [review]

**Comment by dimas-b** [comment]
CI fails beucase of timeouts :( I guess I have to increase them... :hourglass:

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/GraphqlTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6286296&bid=20454288)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/GraphqlTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6286296&bid=20454288)

**Review by dougwettlaufer — APPROVED** [review]

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/GraphqlTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6286296&bid=20454288)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/GraphqlTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6286296&bid=20454288)

**Review by dougwettlaufer — APPROVED** [review]

**Pull request merged**